src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -49,6 +50,49 @@
     private static final ColibriStatsExtension EMPTY_STATS
         = new ColibriStatsExtension();
 
+    /**
+     * A conservative estimate of the average sum of egress + egress packet rate
+     * (in pps) of a video stream flowing through the bridge. Note that the term
+     * "video stream" comes from {@link ColibriConferenceImpl#trackVideoChannelsAddedRemoved},
+     * so it's equivalent to a participant.
+     */
+    private static final int AVG_VIDEO_STREAM_PACKET_RATE_PPS;
+
+    /**
+     * This is the worst case scenario that the bridge must be able to handle
+     * in a 100-peeps call with 20 local senders and 5 local receivers.
+     */
+    private static final double MAX_TOTAL_PACKET_RATE_PPS;
+
+    static
+    {
+        MaxPacketRateCalculator packetRateCalculator = new MaxPacketRateCalculator(
+            4 /* numberOfConferenceBridges */,
+            20 /* numberOfGlobalSenders */,
+            2 /* numberOfSpeakers */,
+            20 /* numberOfLocalSenders */,
+            5 /* numberOfLocalReceivers */
+        );
+
+        MAX_TOTAL_PACKET_RATE_PPS =
+            packetRateCalculator.computeIngressPacketRatePps()
+                + packetRateCalculator.computeEgressPacketRatePps();
+
+        // This currently evaluates to 546. It assumes that the new participant
+        // will be sending video. We're using the "Max" packet rate calculator
+        // so we divide by 3 (an arbitrary factor) to figure out a more
```

Comment:
I don't follow the `3`.  It's just based on assuming a participant will send 1/3rd the max on average?  Is this based on the idea that many people are video muted?

Reply:
Yeah, all that + we assumed a 20 sender call in the packet rate calculation. Maybe it'd be cleaner to assume a smaller call and just take the result of that calculation? Or keep the large call and just take the result of the calculation (no division by 3)

Reply:
Marking this as resolved as it was boomed.